conf/test/titan/grakn-engine.properties

```diff
@@ -18,6 +18,7 @@
 
 #In-Memory or Distributed task manager
 taskmanager.implementation=ai.grakn.engine.tasks.manager.StandaloneTaskManager
+taskmanager.storage.zk=true
```

Comment:
Yep! Cant figure out how to link: It is test `whenEnginePropertiesDoesNotIndicateZookeeperStorage_GraphStorageIsUsed` in `GraknEngineServerTest`